packages/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -30,7 +30,7 @@ export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement>,
   /** @deprecated Use `toggleVariant` instead. Whether or not the dropdown toggle button should have primary button styling */
   isPrimary?: boolean;
   /** Alternate styles for the dropdown toggle button */
-  toggleVariant?: 'primary' | 'secondary' | 'default';
+  toggleVariant?: 'plain-text' | 'primary' | 'secondary' | 'default';
```

Comment:
We usually use camel case for variant names.  Ca we call this on `plainText`?

Reply:
To match the other plain text dropdowns, this would add `isText` that adds `.pf-m-text`, and is used with `isPlain` to create the plain text toggle. ie `<Dropdown isPlain isText />`

Reply:
Good point @mcoker.  I agree.  we should not use the `toggleVariant` and instead add the `isText` prop.

Reply:
Done!